### ISSUE TITLE
using quant files without raw files 

### ISSUE THREAD

**Issue by LindseyOlsen** [vdemichev/DiaNN#789]
The Output section in the README.md says that it is possible to use .quant files without transferring the original raw files. However, when I try to use --use-quant only the files that also have the raw mzML files in the directory will be processed and in the output matrices. If there are no raw files in the directory, then it says 0 files will be processed. 

How can I use only quant files without the raw files for the reanalysis?

I am using the following command 
diann.exe --dir ${LOCAL_DIR} --lib ${LOCAL_DIR}/out-lib.predicted.speclib --threads 92 --verbose 3 --report-lib-info --out ${LOCAL_DIR}/result-out.tsv --qvalue 0.01 --pg-level 1 --mass-acc-ms1 40 --mass-acc 40 --window 9 --int-removal 1 --matrices --temp ${LOCAL_DIR} --smart-profiling --peak-center --use-quant

**Comment by vdemichev** [comment]
When you use --use-quant, DIA-NN will first look for the .quant file with the expected name, and if found will not check if the raw file is present. You can see this by renaming one of the .mzML - this will have no effect. If some other .quant files are not found, check if maybe the typo in the name?

**Comment by LindseyOlsen** [comment]
The quant files names have the pattern of _"path_where_quant_file_was_generanted_mzML_filename".quant. Can I move the quant files to a different directory and then rerun to get the matrices. For example, if I had multiple jobs to create the quant files and then want to rerun them all together.

How does DIANN know what quant files to look for if --dir is used to specify directory of raw files?

**Comment by vdemichev** [comment]
Yes, different directory should be OK, so long as it's specified with the --temp command.
